server/channels/app/platform/busy_test.go

```diff
@@ -24,11 +24,11 @@ func TestBusySet(t *testing.T) {
 
 	require.False(t, busy.IsBusy())
 
-	busy.Set(time.Millisecond * 500)
+	busy.Set(2 * time.Second)
```

Comment:
Recently I sent a PR bumping all timeouts like these to 5 seconds. Let's do it here as well.